pkg/driver/command_driver_test.go

```diff
@@ -1,5 +1,52 @@
 package driver
 
-import "github.com/deislabs/cnab-go/driver"
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"runtime"
+	"testing"
+
+	"github.com/deislabs/cnab-go/driver"
+)
 
 var _ driver.Driver = &CommandDriver{}
+
+func TestCheckDriverExists(t *testing.T) {
+	name := "missing-driver"
+	cmddriver := &CommandDriver{Name: name}
+	if cmddriver.CheckDriverExists() {
+		t.Errorf("Expected driver %s not to exist", name)
```

Comment:
You should use `t.Fatalf` here to stop the test. Errorf will report and keep running the test.